bodhi/tests/utils.py

```diff
@@ -0,0 +1,31 @@
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+This module contains a useful functions that helps with testing
+"""
+
+import re
+
+
+def prepare_text_for_comparison(text):
+    text = text.replace('\n', '')  # Remove newlines
+    text = re.sub(r' +:', '', text)  # Remove start of wrapped lines
+    text = "".join(text.split())  # Remove whitespaces
+    return text
+
+
+def compare_output(output, expected):
+    output = prepare_text_for_comparison(output)
+    expected = prepare_text_for_comparison(expected)
+    return output == expected
```

Comment:
I suggest making this raise an ```Exception``` when they don't match, rather than returning a ```bool```, so that it's more similar to how ```assertEqual()``` works in the test suite. This is OK too if you prefer it.

Reply:
It depends on what we prefer here.
1. We can use this function inside `assertTrue()` method and let outer method raise exception
2. or we can use `compare_output()` alone and raise AssertionError but in this case `compare_output()` should be renamed to contain assert in the name and this can be confusing because it's not part of unittest module.

Reply:
There's a third option that might be cleaner - we could create a ```BaseClientTestCase``` class that subclasses ```unittest.TestCase``` and defines ```assert_cli_output_similar()``` or some name like that. The server tests already have a unified base class so they can do some fancy common things like this, so it would be consistent with what we have over there.